src/remote.ts

```diff
@@ -541,9 +541,29 @@ export class Remote {
     }
   }
 
+  /**
+   * Format's the --log-dir argument for the ProxyCommand
+   */
+  private async formatLogArg(): Promise<string> {
+    if (!supportsCoderAgentLogDirFlag(this.coderVersion)) {
+      return ""
+    }
+
+    // If the proxyLogDirectory is not set in the extension settings we don't send one.
+    // Question for Asher: How do VSCode extension settings behave in terms of semver for the extension?
```

Comment:
```suggestion
```

There are no checks performed on the extension's version, it just depends on whether the setting exists in the `package.json` that is in that version.  If it were somehow missing I think it would just return undefined but I am not sure.  But someone would have to manually edit the `package.json` for that to happen.  :stuck_out_tongue: